recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "6.3.1" %}
-{% set datumgrid_ver = "1.8" %}
+{% set version = "7.0.0" %}
+{% set datumgrid_ver = "1.0" %}
```

Comment:
should this variable be renamed to `data_ver`?

Reply:
I went ahead and made this change.